hs_collection_resource/templates/pages/collectable_resources_modal.html

```diff
@@ -40,6 +41,7 @@ <h4 class="modal-title">Collectable Resources</h4>
                                 </td>
                                 {% with res.raccess as raccess %}
                                     <td>
+                                        <!-- TODO:DRC #5030 if only we could select owners ahead of time... 1 query per res -->
```

Comment:
@pkdash I don't see any way to pre-select owners since they are embeded in a property of ResourceAccess
https://github.com/hydroshare/hydroshare/blob/5030-collection-performance/hs_access_control/models/resource.py#L194
Am I wrong?

Reply:
@devincowan See my proposed solution provided in the comment below.